minkyu/app.js

```diff
@@ -36,10 +39,40 @@ app.use(express.json());
 app.use(cors());
 app.use(morgan('tiny'));
 
-app.get("/ping", (req,res) =>{
-    res.json({message : "pong"});
+app.get("/pings", (req,res) =>{
+    res.status(201).json({"message" : "pong"});
 });
 
+app.post("/users", async(req, res) =>{
+    const { name, email, password } = req.body;
+
+    await database.query(
+        `INSERT INTO users(
+            name,
+            email,
+            password
+        ) VALUES (?, ?, ?, ?);
+        `,
+        [ name, email, password ]
+    );
```

Comment:
```suggestion
    await database.query(
        `INSERT INTO users(
            name,
            email,
            password
        ) VALUES (?, ?, ?);
        `,
        [ name, email, password ]
    );
```
우리가 인자로 넣어주는 값들은 name, email, password 3개 뿐인데 자리에 맞춰서 SQL코드에 값을 넣어주는 placeholder (`?`)는 4개를 작성해주셨네요! 하나는 지워주셔도 될 것 같습니다 :)